integration-templates/unanet/fixtures/opportunity.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What are those fixtures for?

Reply:
To test the actions. Useful for future iterations